backend/pkg/connect/config.go

```diff
@@ -12,17 +12,30 @@ package connect
 import (
 	"flag"
 	"fmt"
+	"time"
+)
+
+const (
+	defaultConnectTimeout = 15 * time.Second
+	defaultReadTimeout    = 60 * time.Second
+	defaultRequestTimeout = 6 * time.Second
```

Comment:
I think we don't need a extra const given that they are only used within SetDefaults(), but if you feel they are required here, it's also fine for me

Reply:
OK, removed